distribution/lib/Standard/Base/0.0.0-dev/src/Errors/Common.enso

```diff
@@ -623,6 +623,7 @@ type Out_Of_Memory
 
 ## Indicates that an expression cannot be evaluated because somewhere within it,
    a function does not have all required arguments provided.
+@Builtin_Type
```

Comment:
- when I was dreaming about using `DataflowError` in #12761
- for example [in this comment](https://github.com/enso-org/enso/issues/12761#issuecomment-2898301797) I believed error is good because:
   - signals the error
   - keeps location of the error
   - flows thru all type checks
- no need to change any syntax as the language already offers all the support for Error values